handbook/engineering/onboarding/index.md

```diff
@@ -0,0 +1,41 @@
+# Engineering onboarding
+
+Welcome! We're excited to have you join the team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Getting set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete [general onboarding](../../people-ops/onboarding/index.md#general-onboarding-checklist)
+- [Configure your GitHub notifications.](../github-notifications/index.md)
+- Join #dev-announce, #dev-chat, and your team's channel on Slack, as well as any other channels you find interesting. [Team chat documentation](../../communication/team_chat.md#engineering)
+- Set up your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/main/doc/dev/getting-started/index.md). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
+- [Add Sourcegraph as a browser search engine](https://docs.sourcegraph.com/integration/browser_search_engine). Add another entry for our internal instance: `https://sourcegraph.sgdev.org/search?q=%s`. This also ensures you have access to our internal instance.
+- Read through your team's handbook page, to learn about the team and its internal processes.
```

Comment:
```suggestion
- Read through your team's handbook page, to learn about the team and its internal processes.
- Add our handbook as a browser search engine `https://about.sourcegraph.com/search?q=%s`.
```

Maybe this can go to the general doc, but I found it quite useful.

Reply:
Yes, this should be in general onboarding for everyone so not going to include it here.